nixos/modules/services/networking/tedicross.nix

```diff
@@ -0,0 +1,142 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  homeDir = "/var/lib/tedicross";
+  cfg = config.services.tedicross;
+  configJSON = pkgs.writeText "tedicross-settings.json" (builtins.toJSON cfg.config);
+  configYAML = pkgs.runCommand "tedicross-settings.yaml" { preferLocalBuild = true; } ''
+    ${pkgs.remarshal}/bin/json2yaml -i ${configJSON} -o $out
+  '';
+
+in {
+  options = {
+    services.tedicross = {
+      enable = mkOption {
```

Comment:
`enable = mkEnableOption "Whether to enable the TediCross Telegram-Discord bridge service.";` is the preferred way to do this.

Reply:
@pacien I made a mistake, I'm sorry!
https://github.com/NixOS/nixpkgs/blob/2e9e63195bf7e371112e95b743c1263759db6e8b/lib/options.nix#L68
So what I should have suggested is: `enable = mkEnableOption "the TediCross Telegram-Discord bridge service.";` as the "Whether to enable " bit is already there.

Reply:
No problem. This is fixed.